### PR TITLE
Aut'akh Origins and Cultures + Accent Goodies

### PR DESCRIPTION
**Pull request by Geevies**
* Added a culture for Aut'akh, as well as an origin for the various communes. Additionally, added a unique accent for Aut'akh.

| Culture & Origins |
 - Aut'akh
   - Aut'akh Valley
   - Undercity Communes
   - Eridani Underworld Commune
   - Razortail Enclave
   - Unknown Aut'akh Commune (headcanon commune)

descriptions ripped from the wiki, unknown commune still needs a description (i'm a better coder than a writer)

| Accents |

- Aut'akh Dialect
  - A mix of all the unathi accents + a little techno spice
image

### CODE REVIEW

**Comment by alsoandanswer** [comment]
!review

**Review by alsoandanswer — APPROVED** [review]
Code looks fine to me. Just needs to be cleared with lore team

File: code/modules/background/accent/unathi.dm [hunk 1]
```diff
@@ -37,4 +37,16 @@
 	Wastelander accent is the sign of an individual who has grown up without a home or clan that could have taught them an ancestral dialect. The accent is apparent in both \
 	Sinta'Unathi and Sinta'Azaziba."
 	tag_icon = "deep_wastelander"
-	text_tag = "WAS"
\ No newline at end of file
+	text_tag = "WAS"
+
+/datum/accent/autakh
+	name = ACCENT_AUTAKH
+	description = "TO-DO get an actual writer to expand on this A crude mix of Hegemonic, Traditionalist and Wastelander dialects owing to Sinta from all walks of life living in a commune together."
```
Comment (NonQueueingMatt) [inline code/modules/background/accent/unathi.dm:44]: Edit this.

File: code/modules/background/accent/unathi.dm [hunk 2]
```diff
@@ -37,4 +37,16 @@
...
+	tag_icon = "autakh_valley"
+	text_tag = "AUT"
+
+/datum/accent/unathi_dreg
+	name = ACCENT_UNATHI_DREG
+	description = "TO-DO get an actual writer to do this"
```
Comment (NonQueueingMatt) [inline code/modules/background/accent/unathi.dm:50]: Edit this.

**Review by NonQueueingMatt — CHANGES_REQUESTED** [review]

**Comment by CaptainGecko** [comment]
Speaking for the lore team here!

We won't take the Eri-Dreg accent, but the standard Aut'akh accent is good to go! As far as the accent desc goes, here's a slightly modified version:
"_Incorporating both Hegemonic and Traditionalist elements, the Aut'akh accent is recognisable for its lack of accentuation and intonations, both owing to the occasional implants of questionable quality having some influence on speech, and the general isolation of most Aut'akh communes._"

**Comment by CaptainGecko** [comment]
It's me again! Because I can't use or read Git like the dumbass I am I forgot about the description for the "Unknown Aut'akh Commune", so here it is.

"Either due to their fellow Unathi rejecting them and their way of life, or life among other species and cultures feeling too alien when away from Hegemonic space, most Aut'akh inevitably end up forming communes and living among each other in these. Aside from the few notable, larger communes, many more minor communes exist across the Spur."

Also, just talked to Geeves about this so it's probably already going to be handled, but the "Unknown Aut'akh Commune" needs the Solarian citizenship option removed, as they cannot speak Sol Common, and knowledge of the solarian tongue is required to obtain said citizenship.

**Comment by Geevies** [comment]
!review

**Review by NonQueueingMatt — APPROVED** [review]

**Pull request merged**